### PR TITLE
Resolve todo comments related to dropping Python 3.7 support

### PR DESCRIPTION
**Pull request by c-bata**
## Motivation

Follow-up https://github.com/optuna/optuna/pull/5727

## Description of the changes

Resolve TODO comments related to dropping support for Python 3.7.

### CODE REVIEW

File: optuna/importance/_ped_anova/evaluator.py [hunk 1]
```diff
@@ -44,15 +44,9 @@ def filter(self, trials: list[FrozenTrial]) -> list[FrozenTrial]:
...
-        def _quantile(v: np.ndarray, q: float) -> float:
-            cutoff_index = int(np.ceil(q * loss_values.size)) - 1
-            return float(np.partition(loss_values, cutoff_index)[cutoff_index])
-
         cutoff_val = max(
             np.partition(loss_values, min_n_top_trials - 1)[min_n_top_trials - 1],
-            # TODO(nabenabe0928): After dropping Python3.7, replace below with
-            # np.quantile(loss_values, self._quantile, method="inverted_cdf").
-            _quantile(loss_values, self._quantile),
+            np.quantile(loss_values, self._quantile, method="inverted_cdf"),
```
Comment (c-bata) [inline optuna/importance/_ped_anova/evaluator.py:49]: [note] `method` argument is available after Numpy v1.22.0 release.
https://github.com/numpy/numpy/releases/tag/v1.22.0

Comment (nabenabe0928) [reply optuna/importance/_ped_anova/evaluator.py:49]: I did not provide sufficient information, sorry :pray:

Comment (not522) [reply optuna/importance/_ped_anova/evaluator.py:49]: But, we can install `numpy<1.22.0` in Python 3.8, isn't it?

Comment (c-bata) [reply optuna/importance/_ped_anova/evaluator.py:49]: We discussed internally and decided to continue the support for `numpy<1.22.0`. Let me update this PR and then make ready for review.

**Review by nabenabe0928 — APPROVED** [review]
Thank you for the changes, LGTM!

**Review by not522 — APPROVED** [review]
LGTM!

**Pull request merged**